charts/aws-efs-csi-driver/values.yaml

```diff
@@ -10,85 +13,99 @@ image:
+    resources: {}
 
 imagePullSecrets: []
-nameOverride: ""
-fullnameOverride: ""
-
-podAnnotations: {}
-
-resources:
-  {}
-  # We usually recommend not to specify default resources and to leave this as a conscious
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
-
-nodeSelector: {}
-
-tolerations: []
-
-affinity: {}
-
-node:
-  podAnnotations: {}
-  tolerations: []
-
-logLevel: 5
-
-hostAliases:
-  {}
-  # for cross VPC EFS, you need to poison or overwrite the DNS for the efs volume as per
-  # https://docs.aws.amazon.com/efs/latest/ug/efs-different-vpc.html#wt6-efs-utils-step3
-  # implementing the suggested solution found here:
-  # https://github.com/kubernetes-sigs/aws-efs-csi-driver/issues/240#issuecomment-676849346
-  # EFS Vol ID, IP, Region
-  # "fs-01234567":
-  #   ip: 10.10.2.2
-  #   region: us-east-2
-
-dnsPolicy: ""
-dnsConfig:
-  {}
-  # Example config which uses the AWS nameservers
-  # dnsPolicy: "None"
-  # dnsConfig:
-  #   nameservers:
-  #     - 169.254.169.253
 
-serviceAccount:
-  controller:
-    # Specifies whether a service account should be created
-    create: true
-    annotations: {}
-    ## Enable if EKS IAM for SA is used
-    #  eks.amazonaws.com/role-arn: arn:aws:iam::111122223333:role/efs-csi-role
-    name: efs-csi-controller-sa
+## Controller deployment variables
 
 controller:
+  # Specifies whether a deployment should be created
   create: true
+  # Number for the log level verbosity
+  logLevel: 5
   # Add additional tags to access points
   tags: {}
     # environment: prod
     # region: us-east-1
   # Enable if you want the controller to also delete the
   # path on efs when deleteing an access point
   deleteAccessPointRootDir: false
+  podAnnotations: {}
+  resources:
+    {}
+    # We usually recommend not to specify default resources and to leave this as a conscious
+    # choice for the user. This also increases chances charts run on environments with little
+    # resources, such as Minikube. If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+    # requests:
+    #   cpu: 100m
+    #   memory: 128Mi
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+  # Specifies whether a service account should be created
+  serviceAccount:
+    create: true
+    ## Enable if EKS IAM for SA is used
+    #  eks.amazonaws.com/role-arn: arn:aws:iam::111122223333:role/efs-csi-role
+    name: efs-csi-controller-sa
+    annotations: {}
+
+## Node daemonset variables
+
+node:
+  # Number for the log level verbosity
+  logLevel: 5
```

Comment:
while we r here, default should be lower, like 2: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/854

Reply:
Lowered to 2 in b1c989de876b5cc719153f51dbe3a831a0a731ed